pkg/pillar/cmd/zedagent/localinfo.go

```diff
@@ -290,10 +351,24 @@ func prepareLocalInfo(ctx *getconfigContext) *profile.LocalAppInfoList {
 	return &msg
 }
 
+// Return how many increments were made to volume's generation counter
+// by a locally triggered purge operations.
+// localAppCommandsLock should be locked when calling this function!
+func getLocalGenCountForVolume(ctx *getconfigContext, volumeID string) int64 {
+	for _, localCmd := range ctx.localAppCommands {
+		if counter, match := localCmd.VolumeGenCounter[volumeID]; match {
```

Comment:
Is this localCmd persisted across reboots of the device?

Reply:
If so a comment about that would be helpful

Reply:
Yes it is persisted, I will add a comment.

Reply:
done